File: exercicios/java/curso_programacao/src/SomaVetor.java
import java.util.Locale;
import java.util.Scanner;

public class SomaVetor {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US); 
		Scanner sc = new Scanner(System.in);
		
		int N, i;
		double soma, media;
		
		System.out.print("Quantos numeros você vai digitar? ");
		N = sc.nextInt();
		
		double[] vet = new double[N];		
		
		for(i = 0; i < N; i++) {
			System.out.print("Digite um número: ");
			vet[i] = sc.nextDouble();
		}
		
		System.out.println();
		
		System.out.print("VALORES = ");
		for(i = 0; i < N; i++) {
			System.out.print(String.format("%.1f ", vet[i]));
		}
		
		System.out.println();
		
		soma = 0;
		for(i = 0; i < N; i++) {
			soma += vet[i];
		}
		System.out.println("SOMA = " + String.format("%.2f",soma));
		
		media = soma / N;		
		System.out.println("MEDIA = " + String.format("%.2f", media));
		
		sc.close();		
		
	}

}

File: exercicios/python/soma_vetor.py
N = int(input('Quantos numeros você vai digitar? '))
vet: [float] = [0 for x in range(N)]

for i in range(0, N):
    vet[i] = float(input('Digite um número: '))

print()
print('VALORES = ', end='')
for i in range(0, N):
    print(f'{vet[i]:.1f} ', end='')

soma: float = 0
for i in range(0, N):
    soma = soma + vet[i]

print()
print(f'SOMA = {soma:.2f}')

media: float = soma / N

print(f'MEDIA = {media:.2f}')
